CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## 5.0.1 - Unreleased
```

Comment:
I'm surprised the changelog parser didn't complain about this given it's complained about other things.

Reply:
Maybe it just needs something in the date and it doesn't check that it is a date? Or maybe it special cases "unreleased"?